packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -97,6 +97,38 @@ describe('cluster', () => {
     });
   });
 
+
+  test('check exposure of EnableIAMDatabaseAuthentication property in DBCluster', () => {
```

Comment:
This description is a little weird. How about

```suggestion
  test('allows setting the `iamAuthentication` property', () => {
```